packages/agent/conf/demos/dbc/templates/dbc/DBCConferenceAttendee.hbs

```diff
@@ -0,0 +1,19 @@
+{
+"@context": [
+"https://www.w3.org/2018/credentials/v1"
+],
+"type": [
+"VerifiableCredential",
+"DBCConferenceAttendee"
+],
+"expirationDate": "{{{ dateTimeAfterDays 7 }}}",
+"credentialSubject": {
+"firstName": "{{firstName}}",
+"lastName": "{{lastName}}",
+"email": "{{email}}",
+"event": {
+"name": "DBC Conference 2023",
+"date": "2023-06-26"
```

Comment:
Should this date not de provided?

Reply:
Good question, but where should we configure this else? the .hbs file is already a conf file. 
Hidden form  fields would work but is not a nice. Extra parameters in the create-credential-offer route step could be, but is that better than in conf/templates?
Maybe create a separate issue for this as this is beyond the scope of DPP-89.

Reply:
fixed